graphql_compiler/schema_generation/sqlalchemy/schema_graph_builder.py

```diff
@@ -0,0 +1,97 @@
+# Copyright 2019-present Kensho Technologies, LLC.
+import warnings
+
+from graphql.type import GraphQLBoolean, GraphQLFloat, GraphQLString
+import six
+import sqlalchemy.sql.sqltypes as sqltypes
+
+from ...schema import GraphQLDate, GraphQLDateTime, GraphQLDecimal, GraphQLInt
+from ..schema_graph import PropertyDescriptor, SchemaGraph, VertexType
+
+
+# TODO(pmantica1): Add scalar mapping for the following classes: Interval.
+# We do not currently plan to add a mapping for JSON and Binary objects.
+SQL_CLASS_TO_GRAPHQL_TYPE = {
+    sqltypes.BIGINT: GraphQLInt,
```

Comment:
An explanation that I found for classes written in all-caps :

> This category of types refers to types that are either part of the SQL standard, or are potentially found within a subset of database backends. Unlike the “generic” types, the SQL standard/multi-vendor types have no guarantee of working on all backends, and will only work on those backends that explicitly support them by name. That is, the type will always emit its exact name in DDL with CREATE TABLE is issued.

Reply:
Can you add this excerpt together with the documentation link where you found it? Otherwise, we'll say this looks good now, and then 6 months from now we'll have forgotten about it and be confused again.

In general, if you feel the need to explain your code in a review comment, strongly consider adding a comment in the code itself, for the above reason.